docs/pktoken.md

```diff
@@ -0,0 +1,702 @@
+
+#### MFA Cosigner Signature
+
+We have included in a Cosigner signature in this example to show what what one looks like. Given that the Cosigner signature does not not differ much between examples we have omitted it from the other examples. Note that PK Tokens can function without a Cosigner signature, a verifier can choose to require or not require one based on a verifiers security needs.
+
+### GQ Signed Nonce-Commitment PK Tokens - Google Example
+
+This PK Token is for the same Google account as the prior example including the same use of the nonce-commitment, the only difference is that we have replaced Google's RSA signature and protected header with a GQ Signature.
+
+```JSON
+{
+  "payload": {
+    "iss": "https://accounts.google.com",
+    "azp": "992028499768-ce9juclb3vvckh23r83fjkmvf1lvjq18.apps.googleusercontent.com",
+    "aud": "992028499768-ce9juclb3vvckh23r83fjkmvf1lvjq18.apps.googleusercontent.com",
+    "sub": "104852002444754136271",
+    "email": "anon.author.aardvark@gmail.com",
+    "email_verified": true,
+    "at_hash": "7y8qnEw17J6BDoqvz6ydbg",
+    "nonce": "8IpXCsOcYBGcCJmXJMFOpBjz4-kPXwDhYi3hm_DFM_U",
+    "name": "Anonymous Author",
+    "picture": "https://lh3.googleusercontent.com/a/ACg8ocIdbWtaAGFsizjWVh7Q6C-XDBuSoUOpf7d7nGqgNQ-9yHmenNA=s96-c",
+    "given_name": "Anonymous",
+    "family_name": "Author",
+    "iat": 1722717291,
+    "exp": 1722720891
+  },
+  "signatures": [
+    {
+      "protected": {
+      "alg": "GQ256",
+      "jkt": "w0bhEOa9d4qxGtKLGhwySJ2VZtRxPA-0abeIC9C-zPQ",
+      "kid": "eyJhbGciOiJSUzI1NiIsImtpZCI6ImUyNmQ5MTdiMWZlOGRlMTMzODJhYTdjYzlhMWQ2ZTkzMjYyZjMzZTIiLCJ0eXAiOiJKV1QifQ",
+      "typ": "JWT"
+      },
+      "signature": "hAo4...(5504 base64 characters)"
+    },
+    {
+      "protected":       {
+      "alg": "ES256",
+      "extra": "yes",
+      "rz": "656f65b99da5d649ea315a52343add3642f14c7ff8d4ebce8ee33a2f4a4b41e0",
+      "typ": "CIC",
+      "upk": {
+          "alg": "ES256",
+          "crv": "P-256",
+          "kty": "EC",
+          "x": "PnzpEjQZ7bsCl2ZExs7dbFQlVzggv-_t50QuzZZWcoc",
+          "y": "1Z-xC6JZL2eAO57ovFJCstnBcMsOiqsGF1NJLyqq1F4"
+      }
+      },
+      "signature": "dsWL..."
+    }
+  ]
+}
+```
+
+#### GQ Signatures
+
+A [GQ (Guillou and Quisquater) signature](https://link.springer.com/content/pdf/10.1007/0-387-34799-2_16.pdf) is a a [Proof of Knowledge (PoK)](https://en.wikipedia.org/wiki/Proof_of_knowledge) of an RSA signature that keeps the RSA signature secret. It provides the same guarantee that the ID Token was signed by OpenID Provider, but it keeps the original RSA signature secret, preventing the ID Token in the PK Token from being used an ID Token. This works because a standard OpenID Connect-compliant service only accepts RSA signature and so will rejected an ID Token that has a GQ signature instead.
```

Comment:
Extra a in "is a Proof of Knowledge (POK)"

Reply:
*Of a RSA

Reply:
*This works because a standard OpenID Connect-compliant service only accepts RSA signature**s** and so will **reject** an ID Token that has a GQ signature instead.

Reply:
Fixed